src/lisflood/global_modules/output.py

```diff
@@ -174,33 +174,30 @@ def dynamic(self):
                     head, tail = os.path.split(where)
                     if '.' in tail:
                         if option['writeNetcdf']:
-                            # CM mod: write end map to netCDF file (single)
-                            # CM ##########################
-
+                            # write end map to netCDF file (single)
                             try:
                                 writenet(0, eval(what), where, self.var.DtDay, maps, report_maps_end[maps].output_var,
-                                         report_maps_end[maps].unit, 'f4', reportStartDate,
+                                         report_maps_end[maps].unit, 'd', reportStartDate,
```

Comment:
As per previous comment, since data_format is not used in writenet, we need to remove the parameter from signature and function calls.

Reply:
Same as before. Should we make this an option and leave the possibility of having smaller files?

Reply:
yes, having the option to set output files precision is the best